manifests/n/Nuand/bladeRF/2025.10/Nuand.bladeRF.installer.yaml

```
@@ -0,0 +1,33 @@
+# Created with komac v2.14.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.10.0.schema.json
+
+PackageIdentifier: Nuand.bladeRF
+PackageVersion: '2025.10'

```

```suggestion
PackageVersion: '1.10.0'
```
Please update file path to match this change.